website/layouts/partials/hooks/head-end.html

```diff
@@ -0,0 +1,3 @@
+{{ with .Site.Params.algolia_docsearch }}
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />
+{{ end }}
```

Comment:
```suggestion
{{ end }}

```

Reply:
Add the missing space at to end to fix it and to retrigger the CI after the migration to Travis ORG.